app/assets/javascripts/application.js

```diff
@@ -17,28 +15,16 @@
 //= require jquery.gridster
 //= require hidden_values
 //= require select_on_click
-//= require select2
```

Comment:
it is still used for the css, will remove it after adding css loading to webpack.